test/integration/analytics_int_test.go

```diff
@@ -644,6 +644,40 @@ func (suite *AnalyticsIntegrationTestSuite) TestCIAndInteractiveDimensions() {
 	}
 }
 
+func (suite *AnalyticsIntegrationTestSuite) TestAnalyticsPixelOverride() {
+	suite.OnlyRunForTags(tagsuite.Analytics)
+
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	cp := ts.Spawn("config", "set", constants.AnalyticsPixelOverrideConfig, "https://example.com")
+	cp.Expect("Successfully set config key")
+	cp.ExpectExitCode(0)
+
+	time.Sleep(time.Second) // Ensure state-svc has time to report events
+
+	cp = ts.Spawn("--version")
+	cp.Expect("ActiveState CLI")
+	cp.ExpectExitCode(0)
+
+	suite.eventsfile = filepath.Join(ts.Dirs.Config, reporters.TestReportFilename)
+	events := parseAnalyticsEvents(suite, ts)
+	suite.Require().NotEmpty(events)
+
+	// Some events will fire before the config is updated, so we expect to
+	// find at least one event with the new configuration values after the service is restarted.
+	foundCount := 0
+	for _, e := range events {
+		if e.URL == "https://example.com" {
+			foundCount++
+		}
+	}
+
+	// Because the service has already reported some events with the default configuration values,
+	// we expect to find at least one event with the new configuration values after the service is restarted.
+	suite.Greater(foundCount, 1)
+}
```

Comment:
Note as written this would pass even if the state-svc didn't pick up the change because it's also counting events from the main state tool invocation.